lib/team_api/team_member_schema.json

```diff
@@ -0,0 +1,84 @@
+{
+  "team": {
+    "title": "team - [team.member].yml files",
+    "description": "Team member metadata",
+    "$schema": "http://json-schema.org/draft-04/hyper-schema",
+    "type": "object",
+    "properties": {
+      "name": {
+        "type": "string",
+        "description": "Unique name for the team member, derived from their GSA email address. Must match the file basename."
```

Comment:
Would prefer to tilt away from hardcoding 18F/GSA-specific things in here, though perhaps it's a future PR issue.